wp-parsely.php

```diff
@@ -79,10 +79,18 @@ public function __construct() {
         $basename = plugin_basename(__FILE__);
         add_filter('plugin_action_links_' . $basename,
                    array($this, 'add_plugin_meta_links'));
-
-        // inserting parsely code
-        add_action('wp_head', array($this, 'insert_parsely_page'));
-        add_action('wp_footer', array($this, 'insert_parsely_javascript'));
+		
+		// Add AMP tracking if this is an AMP page 
+		$currentURL = $this->get_current_url();
+	  	if ((strpos($currentURL, '/amp') !== false) || (strpos($currentURL,'amp=1') !== false)) {
+			// This is an AMP page; add AMP tracking
+		  	insert_parsely_amp_tracking();
+		} else {
+			// This is not an AMP page; do not add AMP tracking
+        	// inserting parsely code
+        	add_action('wp_head', array($this, 'insert_parsely_page'));
+        	add_action('wp_footer', array($this, 'insert_parsely_javascript'));
+        }
```

Comment:
We'll actually want to get rid of the new code added on lines 82-90, removing the `if` block and reverting this section to the way it appeared previously. We'll then simply add the line `add_action('pre_amp_render_post', array($this, 'parsely_add_amp_actions'));` below the two existing `add_action` calls.